docs/src/Upsell.doc.js

```diff
@@ -158,38 +414,19 @@ card(
       width: 128,
     }}
 />
-`}
-  />,
-);
-
-card(
-  <Example
-    name="Link Call to Actions"
-    description={`
-      When \`href\` is supplied to \`primaryAction\` and \`secondaryAction\` the action button defaults to a link-role button.
-    `}
-    defaultCode={`
-<Upsell
-  title="Join the Verified Merchant Program"
-  message="Apply to the Verified Merchant Program—it’s free"
-  primaryAction={{href: "https://pinterest.com", label:"Apply now"}}
-  secondaryAction={{href: "https://pinterest.com", label:"Learn more"}}
-  dismissButton={{
-    accessibilityLabel: 'Dismiss banner',
-    onDismiss: ()=>{},
-  }}
-/>
-`}
-  />,
-);
+        `}
+      />
+    </MainSection.Subsection>
+    <MainSection.Subsection
+      title="Actions"
+      description={`
+      Upsells can have either one primary action, or a primary action and a secondary action. These actions can be Buttons, when no \`href\` is supplied, or Links, by specifying the \`href\`  property.
```

Comment:
`These actions can be buttons, when no \`href\` is supplied, or links, by specifying the \`href\`  property.`